src/configuration.rs

```diff
@@ -0,0 +1,41 @@
+//! This module contains any compile-time configuration parameters for the Stabilizer firmware.
+
+/// MQTT broker IPv4 address
+///
+/// In the default configuration, the IP address is defined as 10.34.16.10.
+pub const MQTT_BROKER: [u8; 4] = [10, 34, 16, 10];
+
+/// Sampling Frequency
+///
+/// Define the frequency at which ADCs (and DACs) are sampled at.
+///
+/// # Units
+/// The units of this parameter are specified as a logarithmic number of ticks of the internal
+/// timer, which runs at 100MHz.
```

Comment:
```suggestion
/// timer, which runs at 100 MHz.
```

All quantities with units get a space between the number and the unit.

Reply:
The other instances in this file should get the space as well.